### ISSUE TITLE
Unable to tab to dropdown for setting with enum values

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#167599]
Testing #167084

1. Search for @lang:go editor spaces
2. Tab to the settings list and tab twice on editor.defaultFoldingRangeProvider 
3. :bug: tab focus jumps to the status bar, I'd expect it to jump to the enum dropdown

**Comment by rzhao271** [comment]
This bug is interesting. If I press down up and then tab for step 2, I can then tab into the dropdown. If I just tab again, it goes to the status bar because it thinks the user wants to focus out of the entire listview and go to the next element, which happens to be the status bar.

In other words, when the user tabs from the Settings editor search bar to the first element in the listview, and presses tab again, we need to decide between focusing a child element of the first element in the listview, or navigating to another part of the editor.

Do you have ideas on what the tabbing behaviour should be in this case? I'm thinking that I can keep focus in the listview, and make it so that the user has to enable "Tab Moves Focus" to go to the next part of the editor, but I'd like to confirm. CC @isidorn

**Comment by meganrogge** [comment]
@rzhao271 that suggestion sounds good then we can see what the community thinks of it

**Comment by rzhao271** [comment]
Unable to repro now
